Documentation/Configuration/Index.rst

```diff
@@ -4,10 +4,10 @@
 Configuration
 =============
 
-In order to use this extension, you must configure which places of worship you
-intend to use. This is done by editing your site configuration
-(stored in :file:`config/sites/<site>/config.yaml`), and adding a list of
-theodia calendars.
+In order to use this extension, you must configure an API key, and which places
+of worship you intend to use. This is done by editing your site configuration
+(stored in :file:`config/sites/<site>/config.yaml`), and adding the API key that
+was given to you, and adding a list of theodia calendars.
```

Comment:
As said in the ticket, I'm missing clear information on how to get that API key. I was not given any key so far and it will not happen automagically without someone requesting it. The documentation should detail how to proceed, where to ask, who to contact, ...